service/storage_proxy.cc

```diff
@@ -4332,7 +4332,7 @@ void storage_proxy::send_to_live_endpoints(storage_proxy::response_id_type respo
 {
     // extra-datacenter replicas, grouped by dc
     std::unordered_map<sstring, host_id_vector_replica_set> dc_groups;
-    std::vector<std::pair<const sstring, host_id_vector_replica_set>> local;
+    std::vector<host_id_vector_replica_set> local;
```

Comment:
The patch description said

> As a reward for our efforts, ... , and we can convert it [the vector with replica sets] to a small_vector, ...

which doesn't happen. Is it a TODO comment, or was this change accidentally lost?